packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -106,9 +105,8 @@ const PersonalDetails = ({
         if (is_qualified_for_idv) {
             idv_error = validateIDV(values);
         }
-        const { errors, warnings } = splitValidationResultTypes(validate(values));
```

Comment:
As confirmed during the meeting, the client can proceed with account creation only if TIN format is correct.